scripts/error-extractor/index.js

```diff
@@ -0,0 +1,86 @@
+/**
+ * Copyright (c) 2013-present, Facebook, Inc.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ */
+'use strict';
+
+var path = require('path');
+var traverse = require('babel-traverse').default;
+
+var evalToString = require('./evalToString');
+var getFilePaths = require('./getFilePaths');
+var invertObject = require('./invertObject');
+var parseJSFile = require('./parseJSFile');
+var writeJSON = require('./writeJSON');
+
+var outputFileName = 'ErrorMap.json';
+var outputFilePath = path.join(__dirname, '../../', outputFileName);
+
+var existingErrorMap;
+var currentID;
+
+try {
+  existingErrorMap = require(outputFilePath);
+} catch (e) {
```

Comment:
(Just wanted to note that this could catch all sorts of errors, like invalid JSON or permissions errors in addition to the file not existing. We probably want to fail silently for the last case and it would be a little better to let the error be thrown in the other cases – but it also doesn't matter that much so we can leave this as-is.)